languages/elixir/exercises/concept/errors/.docs/after.md

```diff
@@ -0,0 +1,72 @@
+## Errors
+
+While Elixir programmers often say ["let it crash"][let-it-crash] and code for the ["happy path"][happy-path], there are often times we need to rescue the function call to return a specific value, message or release system resources.
+
+- Functions that raise errors under normal circumstances are to have `!` at the end of their name.
+- Functions may also alternatively return atoms and tuples to indicate success and failure:
+
+  ```elixir
+  Map.fetch(%{a: 1}, :a)
+  # => {:ok, 1}
+  Map.fetch(%{a: 1}, :b)
+  # => :error
+
+  Map.fetch!(%{a: 1}, :b)
+  # => raises KeyError
+  ```
+
+## Try .. Rescue
+
+- [`try .. rescue`][docs-try] can be used to capture and evaluate errors that are raised inside the block.
+- Let's review the previous example:
+
+  ```elixir
+  try do                             #1
+    raise RuntimeError, "error"      #2
+  rescue
+    e in RuntimeError -> :error      #3
+  end
+  ```
+
+  - **Line 1**, the block is declared with `try`
+  - **Line 2**, the function call which may error is placed here, in this case we are calling `raise/1`
+  - **Line 3**, in the `rescue` section, we pattern match on the _Module_ name of the error raised
+    - on the left side of `->`:
+      - `e` is matched to the error struct
+      - `in` is a keyword in this occurrence
+      - `RuntimeError` is the module to matched, but can match on any error module, or `_` all errors.
+    - on the right side:
+      - the instructions to be executed if the error matches
+
```

Comment:
I kinda expected this section to mention `after` and `else`, what do you think?

Reply:
I think this exercise is about as big as I want it to be, but I was planning a followup where someone defines their own Exception.  Maybe I can mention it there?

Reply:
Either way, I think I will merge this and if we want it in this exercise can amend it later, but I think this will work well in a followup exercise

Reply:
Sure, `after`/`else` are not crucial here, so they might as well be mentioned later.